airflow/www/views.py

```diff
@@ -895,6 +895,8 @@ def _iter_parsed_moved_data_table_names():
                 search=escape(arg_search_query) if arg_search_query else None,
                 status=arg_status_filter if arg_status_filter else None,
                 tags=arg_tags_filter if arg_tags_filter else None,
+                sorting_key=arg_sorting_key,
```

Comment:
Check the pattern above for tags, search and status, we should be consistent (ternary).

Reply:
Thanks for the catch. Added it.